index.md

```diff
@@ -4,113 +4,117 @@ layout: essential
+Here's a subset of tasks you can perform on each host running Cockpit:
 
-### Self-contained
-You don't have to worry about setting up a webserver just to use Cockpit.
+{:.using-cockpit}
+- Inspect and change network settings
+- Configure a firewall
+- Manage storage (including RAID and LUKS partitions)
+- Create and manage virtual machines
+- Download and run containers
+- Browse and search system logs
+- Inspect a system's hardware
+- Upgrade software
+- Keep tabs on performance
+- Manage user accounts
+- Inspect and interact with systemd-based services
+- Use a terminal on a remote server in your local web browser
+- Switch between [multiple Cockpit servers]({{ site.baseurl }}/guide/latest/feature-machines.html)
+- Extend Cockpit's functionality by installing a growing list of apps and add-ons
+- [Write your own custom modules]({{ site.baseurl }}/blog/cockpit-starter-kit.html) to make Cockpit do anything you want
 
-### Uses existing APIs
-Cockpit uses APIs that already exist on the system. It doesn't reinvent subsystems or add a layer of its own tooling.
+*[RAID]: Redundant Array of Inexpensive Disks
+*[LUKS]: Linux Unified Key Setup (encryption)
 
-### Efficient
+Also troubleshoot and fix pesky problems with ease:
 
-Cockpit only uses memory and CPU when active. When inactive, there is no extra load on your server.
+{:.using-cockpit}
+- Diagnose network issues
+- Spot and react to misbehaving virtual machines
+- Examine SELinux logs and fix common violations in a click
+- Inspect detailed metrics that correlate CPU load, memory usage, network activity, and storage performance with the system's journal
 
----
+More features appear in Cockpit every release.
 
-## Glanceable
+### Designed & tested
 
-### System overview
-Immediately understand the health of your server. Cockpit's overview page shows current statistics and the status of your system.
+Cockpit's design keeps your goals in mind.  We test Cockpit with usability studies to make it work the way you'd expect and adjust accordingly. As a result, Cockpit gets easier to use all the time.
 
-### Multi-server
-Monitor and administer [several servers]({{ site.baseurl }}/guide/latest/feature-machines.html) at the same time. Add new hosts and your main server will look after its buddies.
+All code changes have tests which must pass before merging, to ensure stability.
 
-[![Dashboard screenshot]({{ site.baseurl }}/images/site/screenshot-dashboard.png)]({{ site.baseurl }}/images/site/screenshot-dashboard.png)
-{:.screenshot.zoom}
+### Free & free
 
-### Troubleshoot
-Fix pesky problems with ease.
+Cockpit is free to use and [available under the GNU LGPL](https://github.com/cockpit-project/cockpit/blob/master/COPYING).
 
-- Diagnose network issues
-- Spot and react to misbehaving virtual machines
-- Inspect SELinux logs and fix common violations in a click
+### Cockpit works (nearly) everywhere
 
----
+You can install Cockpit on the major distributions, including:
```

Comment:
Let's make this a link to running.html. I know we have several already, but it can't hurt?